src/main/java/seedu/address/logic/parser/UnpoolCommandParser.java

```diff
@@ -18,14 +18,16 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public UnpoolCommand parse(String args) throws ParseException {
+
+        if (args.isBlank() || !args.trim().chars().allMatch(Character::isDigit)) {
```

Comment:
just wondering from a code quality perspective, should we put these 2 conditions as 2 different boolean above the if-else statement for better SLAP